tests/test_data/40aprons.com/fortyaprons.json

```
@@ -0,0 +1,110 @@
+  "language": "en-US",
+  "title": "Raspberry Cocktails with Rosemary",
+  "ingredients": [
+    "1 tablespoon rosemary needles",
+    "1 ½ cups frozen raspberries (defrosted)",
+    "juice of 1 lemon (approximately 2 tablespoons)",
+    "5 tablespoons sugar (¼ cup + 1 tablespoon)",
+    "¾ cup water",
+    "1 tablespoon cornstarch",
+    "1 ½ ounces vodka (or gin)",
+    "½ ounce St. Germaine elderflower liqueur",
+    "1 ½ ounces raspberry-rosemary syrup (made from ingredients above)",
+    "ice",
+    "1 tablespoon soda water",
+    "1 tablespoon champagne",
+    "ice",
+    "rosemary sprigs",
+    "raspberries"
+  ],
+  "ingredient_groups": [
+    {
+      "ingredients": [
+        "1 tablespoon rosemary needles",
+        "1 ½ cups frozen raspberries (defrosted)",
+        "juice of 1 lemon (approximately 2 tablespoons)",
+        "5 tablespoons sugar (¼ cup + 1 tablespoon)",
+        "¾ cup water",
+        "1 tablespoon cornstarch"
+      ],
+      "purpose": "For the Raspberry-Rosemary Syrup (Makes Enough for 6-8 Cocktails)"
+    },
+    {
+      "ingredients": [
+        "1 ½ ounces vodka (or gin)",
+        "½ ounce St. Germaine elderflower liqueur",
+        "1 ½ ounces raspberry-rosemary syrup (made from ingredients above)",
+        "ice",
+        "1 tablespoon soda water",
+        "1 tablespoon champagne"
+      ],
+      "purpose": "For the Cocktail (Makes Enough for 1 Cocktail)"
+    },
+    {
+      "ingredients": [
+        "ice",
+        "rosemary sprigs",
+        "raspberries"
+      ],
+      "purpose": "Serving Suggestions (All Optional)"
+    }
+  ],
+  "instructions_list": [
+    "For the Syrup (Makes Enough for 6-8 Cocktails)",
+    "Add 1 tablespoon rosemary needles to small bowl. Muddle rosemary needles with muddler or back of wooden spoon to release oils.",
+    "Place small saucepan on stovetop over high heat. Add muddled rosemary needles, 1 ½ cups frozen raspberries, juice of lemon, 5 tablespoons sugar, ¾ cup water, and 1 tablespoon cornstarch to saucepan. Stir to incorporate, then bring mixture to rolling boil.",
+    "When mixture begins to boil, immediately reduce heat under saucepan to low.",
+    "Use potato masher to mash raspberries completely, then stir mixture and simmer 10 minutes.",
+    "After 10 minutes, remove saucepan from heat and set aside. Let mixture cool completely.",
+    "Once cooled, pour mixture through fine-mesh strainer or nut-milk bag and into airtight container. Use immediately or refrigerate until ready to use.",
+    "For the Cocktail (Makes Enough for 1 Cocktail)",
+    "Add 1 ½ ounces vodka (or gin), ½ ounce St. Germaine elderflower liqueur, 1 ½ ounces raspberry-rosemary syrup, and ice to cocktail shaker. Shake vigorously.",
+    "Strain cocktail into glass, with or without ice. Add 1 tablespoon soda water and stir to incorporate into drink. Top cocktail with 1 tablespoon champagne, then garnish with rosemary sprigs and raspberries if desired. Serve immediately."
```

```suggestion
    "Strain cocktail into glass, with or without ice. Add 1 tablespoon soda water and stir to incorporate into drink. Top cocktail with 1 tablespoon champagnethen garnish with rosemary sprigs and raspberries if desired. Serve immediately."
```

While your original was correct, the site has a small typo here (that can be seen on the webpage) that excludes the space & comma.